psp/exp_configs/mone.py

```diff
@@ -63,16 +63,18 @@ def get_model(self) -> PvSiteModel:
             ],
             normalize_features=True,
             use_inferred_meta=False,
-            nwp_tolerance=None
+            nwp_tolerance="168h",
```

Comment:
assuming this is the max nwp step to use?

Reply:
It's the max difference in `time` we are willing to tolerate. So in this case if the NWP forecast is more than 7 days old, we'll set it as "missing" (NaN) in the features. Previously we were using the latest NWP forecast even if months old.